app/tree/gui/src/ts/twxplore/gui/geo/reducers/reducerActions.tsx

```diff
@@ -0,0 +1,15 @@
+import reducers from "./reducers"
+
+
+const composedReducer = (state, action) => {
```

Comment:
Wrong file name. What is this for, anyway? Tutorial code?

Reply:
was planning on using redux to communicate with the other components (the selections feature and the featurelist). Going to use dispatch functions to pass the tree selection results.